### PR TITLE
Documentation Enhancement(ecs_task_definition): more clarity on `track_latest`

### PR DESCRIPTION
**Pull request by oycyc**
### Description

Add more clarity for the documentation on the `track_latest`. I was somewhat confused initially reading it and utilizing it, along with others (https://github.com/hashicorp/terraform-provider-aws/issues/20121#issuecomment-2071533920)

This is the original PR https://github.com/hashicorp/terraform-provider-aws/pull/30154

### Relations

This is the original PR https://github.com/hashicorp/terraform-provider-aws/pull/30154

### Output from Acceptance Testing

No tests required. Documentation enhancement

### CODE REVIEW

**Review by ewbankkit — APPROVED** [review]
LGTM :rocket:.

**Comment by ewbankkit** [comment]
@oycyc Thanks for the contribution :tada: :clap:.

**Comment by oycyc** [comment]
Thanks! Seems like I can't merge this, only those with write access can merge.

**Pull request merged**
